base16.go

```diff
@@ -0,0 +1,23 @@
+package multibase
+
+import ()
```

Comment:
:angry:

Reply:
I take it you want it removed.  I don't see the harm, but don't want to make you any angrier. :smile:

Reply:
Fixed.